### ISSUE TITLE
requirements.conda.txt is not in the repo

### ISSUE THREAD

**Issue by mattbrown11** [Kitware/SMQTK#325]
The documentation references requirements.conda.txt, which isn't in the repo.

**Comment by Purg** [comment]
Which documentation file are you looking at?

**Comment by mattbrown11** [comment]
http://smqtk.readthedocs.io/en/latest/building.html#build

**Comment by Purg** [comment]
Right. Basically the last build of that was from about a year or so ago at this point. The "latest" build at that time was broken and there was no auto-rebuild setup on branch update. Currently @kfieldho owns that account so I can't fiddle with it until I pester him to add me to a list to be able to manage it (or some other option).  If you want more up to date docs, follow the instructions for building the doc pages from source, which is basically `pip install sphinx sphinx_rtd_theme` and then run `cd docs && make html`.

**Comment by Purg** [comment]
The main readme for SMQTK could use an update that adds some quick instructions on locally building the docs I suppose.
